kernel/src/hil/framebuffer.rs

```diff
@@ -0,0 +1,143 @@
+//! Interface for FrameBuffer
+use crate::returncode::ReturnCode;
+use enum_primitive::cast::FromPrimitive;
+use enum_primitive::enum_from_primitive;
+
+enum_from_primitive! {
+    #[derive(Copy, Clone, PartialEq)]
+    pub enum ScreenRotation {
```

Comment:
Is there a reason to give these enums numeric values here? I feel like any specific numeric value is too likely to be specific to an implementation to be appropriate for a HIL.

Reply:
I gave them numeric values as they need to be passed from the user space. Any suggestion is welcome.

Reply:
I don't think we want to establish a precedent that a HIL also defines a syscall interface. This mapping can be done in the capsule, much like how GPIO works (with pull up/down and rising/falling edge). This also removes the need to add a dependency to the kernel crate.

Reply:
Got it, solved.